module/x/gravity/types/genesis.go

```diff
@@ -33,7 +33,7 @@ var (
 	ParamsStoreKeyBridgeContractChainID = []byte("BridgeChainID")
 
 	// ParamsStoreKeySignedSignerSetTxsWindow stores the signed blocks window
-	ParamsStoreKeySignedSignerSetTxsWindow = []byte("SignedSignerSetTxWindow")
+	ParamsStoreKeySignedSignerSetTxsWindow = []byte("SignedSignerSetTxsWindow")
```

Comment:
Given that this is an existing item in the param store we can't just change it without migrating data, I was just wondering if this was causing some kind of issue or not.

Reply:
I had forgotten this wasn't a new param, whoops.